lib/FSi/DoctrineExtensions/Mapping/MappedEventSubscriber.php

```diff
@@ -126,23 +119,23 @@ protected function getExtendedMetadataFactory(EntityManagerInterface $entityMana
     }
 
     /**
-     * @param EntityManagerInterface $entityManager
```

Comment:
i think we should either leave the whole docblock if it provides any useful information (as in this case) or remove it completely if it doesn't